app/concepts/ticket/use_case/taken.rb

```diff
@@ -2,8 +2,13 @@ module UseCase
     module Tickets
       class Taken < UseCase::Base::Base
         def call(cinema_hall_id, movie_id)
-          return repository.all.where(cinema_hall_id: cinema_hall_id, movie_id: movie_id).each_with_index.map { |ticket, index| [index, ticket[:seat]] }[0]
```

Comment:
You had `[0]` at the end, so you were taking just a single item from the list. I was suggesting such solution: 

```ruby
repository.all.each_with_index.map { |ticket, index| [index, ticket[:seat]] }.to_h
```

Small explanation:

`[[1, "foo"], [2, "bar"]].to_h # => {1 => "foo", 2 => "bar"}`